projects/lib/package.json

```diff
@@ -14,21 +14,24 @@
     "angular",
     "angular4",
     "angular8",
+    "angular9",
+    "angular10",
     "angular-social-login",
     "social-authentication",
     "social-login",
     "google-authentication",
-    "facebook-authentication"
+    "facebook-authentication",
+    "amazon-authentication"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/abacritt/angularx-social-login/issues"
   },
   "dependencies": {
     "tslib": "^2.0.0"
-},
+  },
   "peerDependencies": {
-    "@angular/common": "^9.1.0",
-    "@angular/core": "^9.1.0"
+    "@angular/common": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0",
+    "@angular/core": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0"
```

Comment:
I am little bit apprehensive about this. I understand that the documentation says that is supports Angular 4+, but my idea was to have a compatibility table with older Angular versions pointing to older releases of this library. The newest version of the library would always support the newest version of Angular.

I am not opposed to this though. I'll be rather happy to merge this if the library still works with all these older versions of Angular.

Reply:
I admit, I am also not fond of this, though it reflects the current setup.
Your suggestion to have it narrowed to one or 2 version of angular per library release is definately preferable and should be the way forward.
I will try and test the latest version with a new angular 4 setup later, to find out what the latest version is that still works with 3.1.1

Reply:
so i tried it out, v2.x works at least down to 5.
3.x seems to need 9 or 10 so i would adjust this to from now on require 9+

Reply:
Thanks, that helps a lot 👍

Reply:
sir i am using Angular 9 and using 3+ version but initially **i got error: "idpiframe_initialization_failed"** error in the console can you please help me?

Reply:
Hi, does anyone know what version works with Angular 4 ? 
2.0.0 doesn't do it

Reply:
I think 1.X versions were made for Angular 4.